### ISSUE TITLE
Reverse obsidian pressure plate

### ISSUE THREAD

**Issue by Draylar** [Draylar/extra-pressure-plates#5]
As requested by a user, add a new pressure plate which is _only_ triggered by mobs or items. Useful for traps.

Potential idea: trapped pressure plate, made with 2 iron bars.

**Comment by mcdodge34** [comment]
Just curious here, is this mod still in dev, its what I started using with my brother in 1.14.2 and I'm worried that we may have to use another mod and replace all our pressure plates everywhere if its a dead mod and one of the feature updates of this game makes it incompatible ???

**Comment by Draylar** [comment]
Yes, it's being maintained. It hasn't been updated because (as far as I know) the 1.14.2 version still works on 1.14.4.

**Comment by mcdodge34** [comment]
You're right about it, so far so good, testing my server that will soon be in 1.14.4 (we abuse some mechanics from 1.14.2 for now) and I'm tracking the mods that haven't been updated for awhile to make sure I won't be in deep trouble, sometimes you end up using a nice little mod and its the one that stop being maintained and its hell on the loose to take all parts of it of your world once it doesn't work with latest version and the dev decided to ditch it up. But speaking of the idea of a mob only pressure plate, that could be very useful.
